docs/source/data.rst

```diff
@@ -1,24 +1,40 @@
 Data
 ====
 
-To facilitate the organization of data, ``ivadomed`` requires the data to be
-organized according to the `Brain Imaging Data Structure (BIDS) <http://bids.neuroimaging.io/>`__ convention.
-An example of this organization is shown below:
+Organization
+------------
+
+To facilitate the organization of data, ``ivadomed`` requires the data to be organized according to the
+`Brain Imaging Data Structure <https://bids.neuroimaging.io/>`_ (BIDS) standard.
+The details of the standard can be found in the `BIDS specification <https://bids-specification.readthedocs.io/>`_.
+
+Validation
+----------
+
+The compliance of the dataset with BIDS can be validate with the `BIDS-validator
+web version <http://bids-standard.github.io/bids-validator>`_.
+Other options for validation are available `here <https://github.com/bids-standard/bids-validator/#quickstart>`_.
+
+Examples
+--------
+
+An example of this organization is shown below for MRI data:
 
 .. image:: https://raw.githubusercontent.com/ivadomed/doc-figures/main/data/1920px-BIDS_Logo.png
     :alt: BIDS_Logo
+    :width: 200
 
 ::
 
     dataset/
     └── dataset_description.json
-    └── participants.tsv
-    └── sub-01  <--------------------- Folder enclosing data for subject 1
+    └── participants.tsv  <--------------------------------- Metadata describing subjects attributes e.g. sex, age, etc.
+    └── sub-01  <------------------------------------------- Folder enclosing data for subject 1
     └── sub-02
     └── sub-03
         └── anat
-            └── sub-03_T1w.nii.gz  <-- MRI image in NIfTI format
-            └── sub-03_T1w.json  <---- Metadata including image parameters, MRI vendor, etc.
+            └── sub-03_T1w.nii.gz  <------------------------ MRI image in NIfTI format
+            └── sub-03_T1w.json  <-------------------------- Metadata including image parameters, MRI vendor, etc.
```

Comment:
```suggestion
            └── sub-03_T1w.json  <------------------------- Metadata including image parameters, MRI vendor, etc.
```
(might as well have all the arrows aligned with the ones below)

Reply:
Yes, that was my intentions, I don't know what happened there 😅.
Fixed in [35296c2](https://github.com/ivadomed/ivadomed/pull/1103/commits/35296c2057da3243b23e7753ae864d6e5813c213)